ambry-coordinator/src/test/java/com.github.ambry.coordinator/MockDataNode.java

```diff
@@ -35,36 +32,16 @@
 public class MockDataNode {
   private DataNodeId dataNodeId;
 
-  private Map<BlobId, MaterializedBlob> blobs;
```

Comment:
The description of that PR says why that change was made.
Let us not complicate matters. You shouldn't have to make any change to `MockDataNode.java` or `Blob.java`. I think you can just hardcode `BlobType.DataBlob` in `MockBlockingChannel.java` as I mentioned in the other comment.